build_manylinux_arm64.sh

```diff
@@ -28,7 +28,7 @@ do
    retVal=$?
    if [ $retVal -ne 0 ]; then
        echo "Error in Tests"
-       exit 1
+#       exit 1
```

Comment:
please revert

Reply:
Updated in [67aa4b5](https://github.com/NREL/pysam/pull/146/commits/67aa4b57c75bd549f0b6ce8c1a498c013456ce8e)